docs/docs-beta/docs/guides/external-systems/apis.md

```diff
@@ -1,4 +1,62 @@
 ---
 title: Connecting to APIs
 sidebar_position: 20
----
\ No newline at end of file
+---
+
+This guide describes how to connect to and interact with APIs in dagster. In this guide you will use dagster Resources to connect to an external API. Using a Resource allows you to standardize how you connect to an external API across your project, use configuration to customize your connections, and use difference implementations of a Resource for different environments.
+
+
+## What you'll learn
+
+- How to write a dagster Resource to connect to an API
+- How to use that Resource in an asset
+- How to configure a Resource
+- How to source configuration values from environment variables
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Familiarity with [Asset definitions](/concepts/assets)
+- Familiarity with [Resources](/concepts/resources)
+- Install the `requests` library: `pip install requests`
+
+</details>
+
+## Step 1: Write a Resource to connect to an API
+
+Here is a minimal Resource for connecting to the an API that returns the sunrise time. In this example, the request URL has been hardcoded to query for data at San Francisco International Airport.
+
+<CodeExample filePath="guides/external-systems/apis/minimal_resource.py" language="python" title="Resource to connect to Sun API" />
+
+
+## Step 2: Use the Resource in an asset
+
+To use the Resource written in Step 1, you can provide it to an asset like this:
+
+<CodeExample filePath="guides/external-systems/apis/use_minimal_resource_in_asset.py" language="python" title="Use the SFOSunResource in an asset" />
+
+
+## Step 3: Configure your Resource
+Many APIs have configuration you can set to customize your usage. Here is an updated version of the Resource from Step 1 with configuration to allow for setting the query location:
+
+<CodeExample filePath="guides/external-systems/apis/configurable_resource.py" language="python" title="Configurable Resource to connect to Sun API" />
+
+## Step 4: Use the configurable Resource in an asset
```

Comment:
Does this need to be a step separate from Step 3? IMO it feels like a continuation of it, otherwise it could be confusing with Step 2

Reply:
also seems like a continuation to me, but i wasn't sure if it would be weird to have the asset usage in its own step in one case but then a sub-set in another case. Happy to make it a sub-set!